Makefile

```diff
@@ -284,6 +327,16 @@ endif
 endif
 endif
 
+# node and v8 use different arch names (e.g. node 'x86' vs v8 'ia32').
+# pass the proper v8 arch name to $V8_ARCH based on user-specified $DESTCPU.
+
+ifeq ($(DESTCPU),x86)
+V8_ARCH=ia32
+else
+V8_ARCH ?= $(DESTCPU)
+
```

Comment:
Style nit: can you drop this newline and the one between the comment and the `ifeq`?